src/include/daos_srv/container.h

```diff
@@ -35,6 +35,7 @@
 #include <daos_srv/pool.h>
 #include <daos_srv/rsvc.h>
 #include <daos_srv/vos_types.h>
+#include <daos_srv/evtree.h>
```

Comment:
I use evt_extent from both lgc_seg->ls_ent_in and phy_ent.pe_rect to calculate the correct ranges for the checksum calculations. This struct is defined in evtree.h, a shared header file for the daos_srv.

Reply:
I meant evt_extent is just a extent of [start, end], it could be replaced by some common neutral structure. It could be better to make csum calculation code independent from evtree. It's fine for me to keep current implementation.

Reply:
I'm keep this include for now due to deadline.  I will rework in follow-on patch to remove dependency on evtree.h.